editor.planx.uk/src/@planx/components/PlanningConstraints/Public.test.tsx

```diff
@@ -62,3 +67,7 @@ it("should not have any accessibility violations", async () => {
   const results = await axe(container);
   expect(results).toHaveNoViolations();
 });
+
+it.todo("fetches classified roads only when we have a siteBoundary"); // using expect(spy).toHaveBeenCalled() ??
+
+it.todo("fetches planning constraints when we have lng,lat or siteBoundary");
```

Comment:
our preview tests for PlanningConstraints leave a lot to be desired, but feel a bit hard to get "right" without a bit more refactoring of this general component (eg to enable easier mocking of siteBoundary & lng, lat content dependencies) - which I'd prefer to revisit as a followup task! stubbed out todos in the meantime to help clarify expected functionality.